discord/message.py

```diff
@@ -804,6 +806,49 @@ def cover(self) -> Optional[Asset]:
         return None
 
 
+class MessageCall:
+    """Represents a message's call data in a private channel from a :class:`~discord.Message`.
+
+    .. versionadded:: 2.5
+
+    Attributes
+    -----------
+    ended_timestamp: Optional[:class:`datetime.datetime`]
+        The timestamp the call has ended.
+    participants: List[Optional[:class:`User`]]
```

Comment:
`List[Optional[X]]` should never be done in the library, so properly filter these. [This is how it was done in the previous implementation](https://github.com/Rapptz/discord.py/blob/1863a1c6636f53592519320a173ec9573c090c0b/discord/message.py#L184-L194).

Reply:
Alright, yeah idk why I thought this would be a good idea 😄 I guess I should also implement the mechanism with searching the users in `self.mentions` for efficiency reasons.